backend/src/core/middlewares/auth.middleware.ts

```diff
@@ -52,6 +52,7 @@ const verifyOtp = async (
         id: user.id,
         createdAt: user.createdAt,
         updatedAt: user.updatedAt,
+        email: user.email,
```

Comment:
Will need to include `email` also in the user object returned by from `AuthService.getUserForApiKey`. Otherwise, calling `GET /settings/email/from` will throw a 500 because `email` is undefined.